### ISSUE TITLE
Faulty PHP DocBlock in Psr.php

### ISSUE THREAD

**Issue by KoenigsKind** [zendframework/zend-log#90]
Faulty PHP DocBlock:
> https://github.com/zendframework/zend-log/blob/5cb1e94d203f3e2c2e848a66ec6888479906d0e8/src/Writer/Psr.php#L58-L59

A possible fix would be:
```
@param  array|Traversable|PsrLoggerInterface $options
@throws \Zend\Log\Exception\InvalidArgumentException
```

### How to reproduce the issue
Open Psr.php in PhpStorm

**Comment by dennybrandes** [comment]
i added right use statement to file.

**Comment by KoenigsKind** [comment]
@dennybrandes 

Nice, could you also change on line 59 the `LoggerInterface` to `PsrLoggerInterface`, as it's declared on line 15?

**Comment by dennybrandes** [comment]
@KoenigsKind 
It is changed.

**Comment by Xerkus** [comment]
Fixed by #93
